.editorconfig

```diff
@@ -13,3 +13,7 @@ max_line_length = off
 trim_trailing_whitespace = false
 indent_style = space
 indent_size = 2
+
+[*.pug]
```

Comment:
Not drastically, but could you explain why you added this?

Reply:
I am using VSCode with the editorconfig plugin, and whenever I edited pug files, it used 4 spaces indentation with tabs instead of spaces. It even changed existing formatting on save.